k-distribution/tests/regression-new/checks/nestedFunctionContextInFun.k.out

```diff
@@ -1,7 +1,11 @@
 [Error] Inner Parser: Parse error: unexpected token '[' following token '('.
 	Source(nestedFunctionContextInFun.k)
 	Location(8,16,8,17)
+	8 |	rule 0 => #fun([[0 => 1]] <bar> 0 </bar>)(0)
+	  .	               ^~
```

Comment:
This doesn't look right.
The error message says only one character is of blame here, but the underline shows two.
The location information for K is an interval that is inclusive on the left, and exclusive on the right: `[0,1)`

Reply:
hey I looked at this closer and i think it's correct in that the error message underlines the location. If there's only one incorrect character, the location should be (8,16, 8, 16) but somehow, the lexer is picking up two characters.

See the following diff which is a part of this PR.

```
diff --git a/k-distribution/tests/regression-new/checks/tokenCheck.k.out b/k-distribution/tests/regression-new/checks/tokenCheck.k.out
index eb2f33f4b1..b3f85ac44c 100644
--- a/k-distribution/tests/regression-new/checks/tokenCheck.k.out
+++ b/k-distribution/tests/regression-new/checks/tokenCheck.k.out
@@ -1,7 +1,11 @@
 [Error] Compiler: Sort X was declared as a token. Productions of this sort can only contain [function] or [token] labels.
        Source(tokenCheck.k)
        Location(10,16,10,21)
+       10 |      syntax X ::= fail()
+          .                   ^~~~~~
 [Error] Compiler: Sort Y was declared as a token. Productions of this sort can only contain [function] or [token] labels.
        Source(tokenCheck.k)
        Location(13,16,13,16)
+       13 |      syntax Y ::= Z
+          .                   ^

```

Reply:
Both of these show terms parsed by the outer parser.
It seems we may have some sort of inconsistency.